src/renderers/shared/shared/event/BrowserEventConstants.js

```diff
@@ -64,6 +64,8 @@ var topLevelTypes = {
   topLoadedMetadata: 'loadedmetadata',
   topLoadStart: 'loadstart',
   topMouseDown: 'mousedown',
+  topMouseEnter: 'mouseenter',
+  topMouseLeave: 'mouseleave',
```

Comment:
why does this file still exist? It seems to be mostly (only?) used as a type import, but has actual code in it. Does this make it into a build? maybe worth changing to a purely flow file so there isn't any worry of runtime usage?